spec/models/piece_spec.rb

```diff
@@ -105,4 +109,118 @@
       end
     end
   end
+
+  describe 'is_obstructed?' do
+    before :all do
+      @game = FactoryGirl.create(:game)
+      @game.pieces.delete
```

Comment:
May need to use delete_all here instead of delete. Double-check with pry, will check when I get home